Monitor/Base.py

```diff
@@ -50,6 +50,12 @@ def parameter(cls, key, default=None):
     # @param algorithm [QCAlgorithm] The algorithm argument that the methods receive is an instance of the base QCAlgorithm class, not your subclass of it.
     # @param targets [List[PortfolioTarget]] The list of targets to be ordered
     def ManageRisk(self, algorithm: QCAlgorithm, targets: List[PortfolioTarget]) -> List[PortfolioTarget]:
+
+        risk_check_times = [time(9, 45, 0), time(10, 15, 0), time(12, 30, 0), time(13, 0, 0), time(13, 30, 0), time(13, 45, 0), time(14, 0, 0), time(15, 0, 0), time(15, 15, 0), time(15, 45, 0)]
+        current_time = self.context.time.time()
+        if current_time not in risk_check_times:
```

Comment:
I don't think we need this as we have the managePositionFrequency below that checks for it. Right now the default for Monitor is 1 minute but we can change that default to be every 30 minutes or less if it improves. But its controllable. https://github.com/Chocksy/qc-options-framework/blob/main/Monitor/Base.py#L13